### PR TITLE
Fixes captain's helmet not showing masks

### LINKED ISSUE #20540: Captain's hardsuit's helmet sprite mutilated
Dear god it just looks so ugly, you can't see the gas mask the captain is wearing and it has a tiny hole in it for no apparent reason.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #20540

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -494,6 +494,7 @@
 	desc = "A special hardsuit helmet designed for only the most fashionable of military figureheads."
 	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
 	actions_types = list()
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR //we want to see the mask
```
Comment (GunHog) [inline code/modules/clothing/spacesuits/hardsuit.dm]: My bad for failing to notice this. When I was testing the change, I assumed it was normal.

Comment (phil235) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Remove HIDEFACIALHAIR as well, the helmet clearly does not hide facial hairs.

**Comment by phil235** [comment]
Hey @KorPhaeron I comment wasn't addressed.

**Comment by KorPhaeron** [comment]
I somehow missed it. Probably shouldn't be merging things on mobile.

I'll make a pr for it when I get home, sorry.

**Comment by Sligneris** [comment]
Something went wrong, and the masks are still being hidden.

**Comment by phil235** [comment]
>I'll make a pr for it when I get home, sorry.

...
> 18 days ago

**Pull request merged**